graphics/jgraph/Portfile

```diff
@@ -3,13 +3,13 @@
 PortSystem          1.0
 
 name                jgraph
-version             8.3
+version             8.5
 categories          graphics
 # see http://packages.debian.org/changelogs/pool/main/j/jgraph/current/copyright
-license             GPL
+license             BSD-3-Clause
 homepage \
     http://www.cs.utk.edu/~plank/plank/jgraph/jgraph.html
```

Comment:
```suggestion
    https://www.cs.utk.edu/~plank/plank/jgraph/jgraph.html
```

Reply:
The site serves a bad cert, hence the http link.

Reply:
sorry, you're right!